### PR TITLE
Added privacy descriptions for camera and PhotoLibrary

### PR DESCRIPTION
**Pull request by tclaus**
As well as localizations for this

### CODE REVIEW

**Comment by tclaus** [comment]
How to connect these (IMHO minor important) strings to weblate? 
For now this should be suficient - at least to not crash the app when accessing photos or camera

File: ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme [hunk 1]
```diff
@@ -53,6 +53,7 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "de"
```
Comment (jhass) [inline ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme]: This is the main language? Let's use `en` here then :)

**Comment by jhass** [comment]
> How to connect these (IMHO minor important) strings to weblate?

Good question. I guess we'll need to add a new component to the Weblate project, since one can define only one base file per component. A bit silly to have multiple components pointing to the same repo I guess, but oh well. But I agree this can wait a bit :)

**Comment by jhass** [comment]
🙇

**Comment by jhass** [comment]
I added a component in the weblate project, using the auto detected settings for iOS. Not sure if that's all correct but we'll figure that out soon enough I guess :) 

https://hosted.weblate.org/projects/insporation/ios/

**Pull request merged**